Tools/python/SampleFiles.py

```diff
@@ -851,6 +861,1655 @@ def who(name, ds):
+'ZHToSSTodddd_tau1mm_M55_20162': (26, ['/store/group/lpclonglived/pkotamni/ZH_HToSSTodddd_ZToLL_MH-125_MS-55_ctauS-1_TuneCP5_13TeV-powheg-pythia8/TrackMoverMCTruthOnnormdzULV30LepMumv6_20162/240705_101410/0000/mctruth_%i.root' % i for i in chain(xrange(2), xrange(1,3), xrange(2,4), xrange(5,7), xrange(8,10), xrange(9,11), xrange(10,12), xrange(11,13), xrange(12,14), xrange(13,15), [0, 3, 5, 6, 8, 14])]),
+'WplusHToSSTodddd_tau1mm_M55_20162': (240, ['/store/group/lpclonglived/pkotamni/WplusH_HToSSTodddd_WToLNu_MH-125_MS-55_ctauS-1_TuneCP5_13TeV-powheg-pythia8/TrackMoverMCTruthOnnormdzULV30LepMumv6_20162/240630_132737/0000/mctruth_%i.root' % i for i in chain(xrange(2), xrange(1,3), xrange(2,4), xrange(3,5), xrange(4,6), xrange(7,9), xrange(8,10), xrange(9,11), xrange(10,12), xrange(11,13), xrange(12,14), xrange(13,15), xrange(14,16), xrange(15,17), xrange(16,18), xrange(17,19), xrange(18,20), xrange(19,21), xrange(20,22), xrange(21,23), xrange(22,24), xrange(23,25), xrange(24,26), xrange(25,27), xrange(26,28), xrange(27,29), xrange(28,30), xrange(29,31), xrange(30,32), xrange(31,33), xrange(32,34), xrange(33,35), xrange(34,36), xrange(35,37), xrange(36,38), xrange(37,39), xrange(38,40), xrange(39,41), xrange(40,42), xrange(41,43), xrange(42,44), xrange(43,45), xrange(44,46), xrange(45,47), xrange(46,48), xrange(47,49), xrange(48,50), xrange(49,51), xrange(50,52), xrange(51,53), xrange(52,54), xrange(53,55), xrange(54,56), xrange(55,57), xrange(56,58), xrange(57,59), xrange(60,62), xrange(61,63), xrange(64,66), xrange(65,67), xrange(66,68), xrange(67,69), xrange(68,70), xrange(69,71), xrange(70,72), xrange(71,73), xrange(72,74), xrange(73,75), xrange(74,76), xrange(75,77), xrange(76,78), xrange(77,79), xrange(78,80), xrange(79,81), xrange(80,82), xrange(81,83), xrange(82,84), xrange(83,85), xrange(84,86), xrange(85,87), xrange(86,88), xrange(87,89), xrange(88,90), xrange(89,91), xrange(90,92), xrange(91,93), xrange(94,96), xrange(95,97), xrange(96,98), xrange(97,99), xrange(98,100), xrange(99,101), xrange(100,102), xrange(101,103), xrange(102,104), xrange(103,105), xrange(104,106), xrange(105,107), xrange(106,108), xrange(107,109), xrange(108,110), xrange(109,111), xrange(110,112), xrange(111,113), xrange(112,114), xrange(113,115), xrange(114,116), xrange(115,117), xrange(116,118), xrange(117,119), xrange(118,120), xrange(119,121), xrange(120,122), xrange(121,123), xrange(122,124), [0, 5, 7, 58, 60, 62, 64, 92, 94, 123])]),
```

Comment:
@pkotamnives Have you checked that using these ntuples works as expected? I am surprised because I see what look to be duplicates, and a simple test with interactive python shows:

```
>>> from itertools import chain
>>> for i in chain(xrange(2), xrange(1,3), xrange(2,4)) :
...     print i
... 
0
1
1
2
2
3
```

Let's make sure we aren't using any duplicate files, and I think it would be good to track down the source of this bug, if it is affecting our results.

Reply:
If this particular set of files is really old and currently unused, let's check that the things we currently use aren't affected by the same thing, and go from there.

Reply:
I'll also note that `'WplusHToSSTodddd_tau1mm_M55_20162': (240,` indicates 240 files for this, but the file indices only go up to ~124 or so. This really looks like we're using duplicates here.